src/com/intellij/idea/plugin/hybris/toolwindow/RemoteSolrConnectionDialog.kt

```diff
@@ -155,4 +223,11 @@ class RemoteSolrConnectionDialog(
         e.message ?: ""
     }
 
+    private fun updateWslIp(distributions: List<WSLDistribution>) {
+        val selected = wslDistributionComboBox.selectedItem as String
+        val wslIp = distributions.find { it.msId == selected }?.wslIpAddress.toString()
+        hostTextField.text = wslIp.replace("/", "")
```

Comment:
possible NPE, please adjust the code (IDE should highlight this possible issue)

Reply:
Completed